dali/operators/image/remap/rotate_params.h

```diff
@@ -138,17 +138,19 @@ class RotateParamProvider
     int s = 0;  // sample index
     int64_t ofs = 0;  // offset within sample
     for (int64_t i = 0; i < n; i++) {
-      out[i] = TL.data[s][ofs++];
       if (ofs == sample_size) {
         ofs = 0;
-        sample_size = TL.shape[++s].num_elements();
+        s++;
+        assert(s < TL.num_samples());
+        sample_size = TL.shape[s].num_elements();
       }
+      out[i] = TL.data[s][ofs++];
```

Comment:
Moved because when `ofs` was increased before checking for end of sample, there was out-of-bounds access to TL.shape.